eng/WpfArcadeSdk/Sdk/Sdk.targets

```diff
@@ -16,6 +16,7 @@
   <Import Project="$(WpfArcadeSdkToolsDir)ExtendedAssemblyInfo.targets" />
   <Import Project="$(WpfArcadeSdkToolsDir)Redist.targets" />
   <Import Project="$(WpfArcadeSdkToolsDir)WpfProjectReference.targets" />
+  <Import Project="$(WpfArcadeSdkToolsDir)CodeAnalysis.targets" />
```

Comment:
Have you tested the build of DotNET-wpf-int with this change?

Reply:
Is this a general question or is it specific to this change (CodeAnalysis.props -> CodeAnalysis.targets)?

Reply:
It's re: changes to `Microsoft.DotNet.Arcade.Wpf.Sdk`

Reply:
There's no reason to believe that this simple change will cause any issues. None of the internal projects are building with `<EnableAnalzers>true</EnableAnalyzers>`, so the build won't start failing due to analyzers being integrated where they weren't before

Reply:
but i'll go ahead and make sure and report back if it's clean

Reply:
it builds fine